4-esm-webpack/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict'
+
```

Comment:
I changed the `package.json` main entry to `index.js` (to more align with the `npm init esm` or `yarn create esm` result.

Reply:
Makes sense!